server/api/user.js

```diff
@@ -12,3 +12,44 @@ router.get('/', requireToken, async (req, res, next) => {
     next(err);
   }
 });
+
+router.post('/', async (req, res, next) => {
+  try {
+    //const user = await User.findByToken... when token available
+    const randomUser = await User.findByPk(1)
+  
+    const randomDoki = ["fox", "cat", "bunny"][Math.floor(Math.random() * 3)];
+    const doki = await Doki.findOne(
+      {where: {
+        type: randomDoki
+      }})
+    await randomUser.addDoki(doki)
+    let userDoki = await User_Doki.findOne(
+      {
+        where: {
+          userId: randomUser.id,
+          dokiId: doki.id
+        }
+      }
+    )
+    await userDoki.set({dokiName: req.body.dokiName})
+    res.send(await userDoki.save())
+  } catch (err) {
+    next(err);
+  }
+})
+
+router.get('/doki',async (req, res, next) => {
```

Comment:
if your route to return a user-doki is `GET /user/doki` then your route to add a user-doki should be `POST /user/doki` to keep your routes consistent